www/front_src/src/Resources/Details/tabs/Services/index.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react';
+
+import { makeStyles, Paper, Typography } from '@material-ui/core';
+
+import { useRequest, StatusChip } from '@centreon/ui';
+import { isNil } from 'ramda';
+import { Skeleton } from '@material-ui/lab';
+import { TabProps } from '..';
+import { listServices } from './api';
+import { listServicesDecoder } from './api/decoders';
+import { Service } from './models';
+import { useResourceContext } from '../../../Context';
+
+const useStyles = makeStyles((theme) => ({
+  services: {
+    display: 'grid',
+    gridAutoFlow: 'row',
+    gridGap: theme.spacing(1),
+    width: '100%',
+  },
+  service: {
+    display: 'grid',
+    gridAutoFlow: 'columns',
+    gridTemplateColumns: 'auto 1fr auto',
+    padding: theme.spacing(1),
+    gridGap: theme.spacing(2),
+    alignItems: 'center',
+  },
+  description: {
+    display: 'grid',
+    gridAutoFlow: 'row',
+    gridGap: theme.spacing(1),
+  },
+}));
+
+const LoadingSkeleton = (): JSX.Element => {
+  const classes = useStyles();
+
+  const serviceLoadingSkeleton = (
+    <div className={classes.service}>
+      <Skeleton variant="circle" width={25} height={25} />
+      <Skeleton height={25} />
+      <Skeleton width={50} height={25} />
+    </div>
+  );
+
+  return (
+    <div className={classes.services}>
+      {serviceLoadingSkeleton}
+      {serviceLoadingSkeleton}
+      {serviceLoadingSkeleton}
```

Comment:
Why 3 same occurences ?

Reply:
these are loading skeletons, we could display 2 of them or even 4, I just thought 3 might be enough